packages/auth-client/src/types/client.ts

```diff
@@ -60,7 +60,7 @@ export abstract class IAuthClient {
   public abstract pairingTopics: IStore<string, { topic: string; pairingTopic: string }>;
   public abstract requests: IStore<
     number,
-    { id: number } & (AuthEngineTypes.Cacao | AuthEngineTypes.PendingRequest)
+    { id: number; topic: string } & (AuthEngineTypes.Cacao | AuthEngineTypes.PendingRequest)
```

Comment:
The inline `topic` (or rather `pairingTopic`) property here seems to be redundant because `AuthEngineTypes.PendingRequest` is already being expanded here: https://github.com/WalletConnect/auth-client-js/pull/71/files#diff-13343927fa29a572c4153af18e4a3f7e7e3bc914b5f5c758f21fd31f5fe362d1R88